index.php

```diff
@@ -0,0 +1,169 @@
+<?php include("php/db.php"); ?>
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="css/flowbite.min.css" rel="stylesheet" />
+    <link rel="stylesheet" href="css/style.css">
+    <title>Home</title>
+</head>
+<body>
+    <!-- Navbar -->
+    <?php include("header.php"); ?>
```

Comment:
Errors are outlined in header.php, but the navbar text colours aren't correct for some links.